ActindoWorkFlow/UserInterface/05a_QueueTypes.md

```diff
@@ -1,32 +1,34 @@
 [!!Configure the queue types](../Integration/01_ConfigureQueueTypes.md)
 
-# Overview
 
-*Workflows > Queue types > Tab OVERVIEW*
+# OVERVIEW (Queue types)
+
+*Process Orchestration > Queue types > Tab OVERVIEW*
 
 ![Queue types](../../Assets/Screenshots/ActindoWorkFlow/QueueTypes/Overview.png "[Queue types]")
 
-**Queue type list**
+**Queue types**
+
+The list displays all queue types. All fields are read-only. Depending on the settings, the displayed columns may vary. Click a queue type in the list to display the *Edit queue type* view.    
 
 - [KILL WORKER]   
-    Click this button to stop all workers that are currently executing actions in all processes.   
+    Click this button to stop all workers that are currently executing actions in all processes.  
 
-- ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh)   
-    Click this button to update the list of queue types.
+- [START WORKER]   
+    Click this button to start all workers again, for example, if you have previously stopped all workers.
 
-- ![Columns](../../Assets/Icons/Columns.png "[Columns]") Columns (x)   
-    Click this button to display the columns bar and customize the displayed columns and the order of columns in the list. The *x* indicates the number of columns that are currently displayed in the list.
+The following functions are available for the editing toolbar:
 
 - [x]      
     Select the checkbox to display the editing toolbar.
 
 - [DELETE]   
-    Click this button to delete the selected queue type. The *Queue type has been deleted* pop-up window is displayed. The *Default* queue type cannot be deleted.  
+    Click this button to delete the selected queue type. A confirmation message is displayed. The *Default* queue type cannot be deleted.  
```

Comment:
Vorschlag: Click this button to delete the selected queue type. When a queue type has been deleted, a confirmation message is displayed. 
> [Info] The *Default* queue type cannot be deleted. / Bear in mind that the *Default*... cannot be deleted.

(Ich finde, so ist es deutlicher)